modules/persistent-storage-csi-azure-file-cross-sub-dynamic-pre-provisioning-pv-pvc-procedure.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent_storage-csi-azure-file.adoc
+//
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-azure-file-cross-sub-dynamic-pre-provisioning-pv-pvc-procedure_{context}"]
+= Pre-provisioning across subscriptions for Azure File by creating a PV and PVC:
```

Comment:
It's usually called static provisioning rather than pre-provisioning
@RomanBednar @duanwei33 thoughts?

Reply:
In my understanding, these two words can be used interchangeably for the mentioned scenario. Sure, “static provisioning” is okay to me.

Reply:
I think in our content, we are generally using the term "static", so to be consistent, I'll change it to static.
\